src/server/rest/v1/service/TagService.ts

```diff
@@ -562,4 +643,24 @@ export default class TagService {
     );
     return result;
   }
+
+  private static async importTag(action: ServerAction, req: Request, tag: any): Promise<void> {
+    try {
+      const newUploadedTag: ImportedTag = {
+        id: tag.id.toUpperCase(),
+        description: tag.description ? tag.description : `Tag ID '${tag.id}'`,
+        importedBy: req.user.id,
+        status: ImportStatus.UNKNOWN
```

Comment:
Use a specific status like TO_BE_IMPORTED or NEWLY_IMPORTED that should be set when created in the collection

Reply:
removed the status since we have only two cases